README.md

```diff
@@ -34,115 +63,157 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
     • [CHANGELOG](https://github.com/dahlbyk/posh-git/blob/v0.7.0/CHANGELOG.md) )
 
 ## Notes
+
 posh-git adds variables to your session to let you customize it, including `$GitPromptSettings`, `$GitTabSettings`, and `$TortoiseGitSettings`.
 For an example of how to configure your PowerShell profile script to import the posh-git module and create a custom prompt function that displays git status info, see the [Customizing Your PowerShell Prompt](https://github.com/dahlbyk/posh-git#step-3-optional-customize-your-powershell-prompt) section below.
 
 Note on performance: Displaying file status in the git prompt for a very large repo can be prohibitively slow.
 Rather than turn off file status entirely (`$GitPromptSettings.EnableFileStatus = $false`), you can disable it on a repo-by-repo basis by adding individual repository paths to `$GitPromptSettings.RepositoriesInWhichToDisableFileStatus`.
 
 ## Installation
+
 ### Prerequisites
+
 Before installing posh-git make sure the following prerequisites have been met.
 
-1. PowerShell 3.0 or higher. Check your PowerShell version by executing `$PSVersionTable.PSVersion`.
+1. Windows PowerShell 5.x or PowerShell Core 6.0.
+   Get PowerShell Core 6.0 for Windows, Linux or macOS from [here][pscore-install].
+   You can check your PowerShell version by executing `$PSVersionTable.PSVersion`.
 
-2. Script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
+1. On Windows, script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
```

Comment:
Can these Markdown lint rules be tuned? One of the key objectives of Markdown is to be readable as plain text, so I prefer to use real numbering.

I also prefer inline links over reference links, especially for the CHANGELOG issue/PR links, though that preference isn't particularly strongly held.

Thoughts?

Reply:
As long as the linter won't complain about preferred style on all our (two) markdown files, I'm good. Reference links on readme is fine.

Reply:
Updated the settings to not complain about numbered lists 1., 2., 3., etc and reverted the numbering change.